README.md

```diff
@@ -1,2 +1,39 @@
 # Awesome Cosmonic
-Awesome projects and source code for the Things To Build section on https://cosmonic.com/docs/category/things-to-build
+Awesome projects and source code for the Things To Build section on https://cosmonic.com/docs/category/things-to-build.
+
+| Deployment  | Link | Topology |
+| ------------- | ------------- | ------------- |
+| XKCD Generator on Cosmonic  | [<img src="https://cosmonic.com/badge/deploy.svg" alt="Deploy on Cosmonic" width="400">](https://new.cosmonic.app/?yaml=https://raw.githubusercontent.com/cosmonic/awesome-cosmonic/main/xkcdgenerator/xkcd-generator-cosmonic.wadm) | This example will deploy the XKCD actor, HTTP Client Provider, and HTTP Server Provider all on Cosmonic.  |
+| XKCD Generator on Stargate  | [<img src="https://cosmonic.com/badge/deploy.svg" alt="Deploy on Cosmonic" width="400">](https://new.cosmonic.app/?yaml=https://raw.githubusercontent.com/cosmonic/awesome-cosmonic/main/xkcdgenerator/xkcd-generator-stargate.wadm) | This example will deploy the XKCD actor & HTTP Client Provider on a remote startgate node, and the HTTP Server Provider on Cosmonic. In order for this manifest to work you will need at least one stargate host deployed.  |
+
+# How does it work?
+One of the powerful things about Cosmonic is that your basic application architecture can be deployed to dramatically different application topologies all without changing or modifying your architecture.  In the simple examples below we use [Open Application Manifest WADM Manifests](https://github.com/wasmCloud/wadm) to automatically deploy applications to Cosmonic.
+
+Some manifests require multiple Cosmonic hosts connected to your Constellation. Please follow the [Cosmonic Getting Started Guide](https://cosmonic.com/docs/user_guide/cli/getting_started) to deploy a host or simply:
+
+```bash -c "$(curl -fsSL https://cosmonic.sh/install.sh)"```
+
+## Host Labels
+Just like Kubernetes Cosmonic uses labels as a powerful mechanism for deployments.  There are a handful of labels that are automatically defined by default for you depending on where the host is executing:
```

Comment:
Comma between Kubernetes and Cosmonic

You say powerful twice too

Reply:
rewritten.